src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -240,6 +243,18 @@ private String createTimerName(final HttpRequest request) {
         return nameBuilder.toString();
     }
 
+    private String createRequestSizeCounterName(final HttpRequest request) {
+        final StringBuilder nameBuilder = new StringBuilder()
+                .append("rest_service/")
+                .append(request.method().value());
+        if (!request.getUri().path().startsWith("/")) {
+            nameBuilder.append("/");
+        }
+        nameBuilder.append(request.getUri().path());
```

Comment:
Seems like this is identical to the timer one above; refactor? Also, I don't think having a path be a metric and folder in ReMet is going to look the best. I think we should be explicit wrt latency.